.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+.db3
```

Comment:
There is an extra space before the `.db3` entry. This might prevent Git from ignoring `.db3` files as intended. Consider removing the leading space.